stream_alert_cli/helpers.py

```diff
@@ -146,19 +146,20 @@ def _create_lambda_function(function_name, region):
         MemorySize=128,
         Publish=True,
         Code={
-            'ZipFile': _make_lambda_package()
+            'ZipFile': make_lambda_package()
         }
     )
 
-def _encrypt_with_kms(data, region, alias):
+
+def encrypt_with_kms(data, region, alias):
     kms_client = boto3.client('kms', region_name=region)
     response = kms_client.encrypt(KeyId=alias,
                                   Plaintext=data)
 
     return response['CiphertextBlob']
 
 
-def _make_lambda_package():
+def make_lambda_package():
```

Comment:
This one isn't used outside of this helper file (not imported elsewhere) and could be kept private. It's only used by the `create_lambda_function` function.